### PR TITLE
Contaocamp23 onboarding

### PR DESCRIPTION
**Pull request by fiedsch**
New Version of #1199 which is closed because I messed things up. 

Please also see the comments there!

Current question to be decided: should we use the long or the long or the short version of the text?

### CODE REVIEW

**Comment by ausi** [comment]
I’d favor the long version.

File: docs/manual/installation/quickstart.de.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+title: "Contao Installieren – Quickstart"
+description: "Die ersten Schritte zur Contao Installation mit dem Contao Manager."
+url: "installation/quickstart"
+aliases:
+- /de/installation/quickstart/
+weight: 90
+hidden: true
+---
+
+Wir gehen hier davon aus, daß du entweder die aktuellste Version oder die Long Term Support Version mit dem Contao Manager
```
Comment (rabauss) [inline docs/manual/installation/quickstart.de.md]:
```suggestion
Wir gehen hier davon aus, dass du entweder die aktuellste Version oder die Long Term Support Version mit dem Contao Manager
```
Neue Rechtschreibung

File: docs/manual/installation/quickstart-checklist.de.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+title: "Contao Installieren – Quickstart"
+description: "Die ersten Schritte zur Contao Installation mit dem Contao Manager."
+url: "installation/quickstart-checklist"
+aliases:
+- /de/installation/quickstart-checklist/
+weight: 90
+hidden: true
+---
+
+Wir gehen hier davon aus, dass du entweder die aktuellste Version oder die Long Term Support Version mit dem Contao Manager
```
Comment (netzarbeiter) [inline docs/manual/installation/quickstart-checklist.de.md]:
```suggestion
Wir gehen hier davon aus, dass du entweder die aktuellste Version oder die [Long Term Support Version](https://docs.contao.org/manual/de/installation/contao-aktualisieren/#long-term-support-versionen) mit dem Contao Manager 
```

File: docs/manual/installation/quickstart-checklist.de.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+weight: 90
+hidden: true
+---
+
+Wir gehen hier davon aus, dass du entweder die aktuellste Version oder die Long Term Support Version mit dem Contao Manager 
+installieren willst. Das ist der einfachste und für Einsteiger empfohlene Weg.
+
+## Hosting Konfiguration
+
+Das Hosting konfigurierst Du über das Admin-Panel deines Hosting-Providers.
```
Comment (netzarbeiter) [inline docs/manual/installation/quickstart-checklist.de.md]:
```suggestion
Das Hosting konfigurierst du über das Admin-Panel deines Hosting-Providers.
```

File: docs/manual/installation/quickstart-checklist.de.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+In Contao befinden sich alle öffentlich erreichbaren Dateien im Unterordner `/public` der Installation. Erstelle
+dazu den Ordner `public` und setze das Wurzelverzeichnis (Document Root) der Installation über das Admin-Panel
+des Hosting-Providers auf diesen Unterordner.
+
+**Beispiel:** `example.com` zeigt auf das Verzeichnis `/www/example/public`
+
+### Datenbank
+
+Contao benötigt für den Betrieb eine Datenbank 
+([MySQL](../../installation/systemvoraussetzungen/#mysql-mindestanforderungen)), die Du idealerweise auch gleich
```
Comment (netzarbeiter) [inline docs/manual/installation/quickstart-checklist.de.md]:
```suggestion
([MySQL](../../installation/systemvoraussetzungen/#mysql-mindestanforderungen)), die du idealerweise auch gleich 
```

File: docs/manual/installation/quickstart-checklist.de.md [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+Bevor du nun Contao installierst, muss der Manager selbst konfiguriert werden. Lege dazu einen neuen Benutzer an, indem
+du einen Benutzernamen und ein Passwort vergibst. Benutzername und Passwort sind unabhängig von der späteren Contao Installation!
+
+
+### Serverkonfiguration
+
+Der Contao Manager benötigt den Pfad zum PHP-Binary und weitere Server-Informationen, um Hintergrund-Prozesse korrekt
+auszuführen. In der Regel werden diese Informationen automatisch erkannt.
+
+Details zu diesem Schritt findest Du [hier](../contao-manager/#serverkonfiguration).
```
Comment (netzarbeiter) [inline docs/manual/installation/quickstart-checklist.de.md]:
```suggestion
Details zu diesem Schritt findest du [hier](../contao-manager/#serverkonfiguration).
```

File: docs/manual/installation/quickstart-checklist.de.md [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+muss das [Contao-Installtool](../contao-installtool/) gestartet werden (Contao 4.13 LTS) bzw. die benötigten Informationen 
+direkt im Contao Manager erfasst werden (Contao 5.x). 
+
+
+## Ein Administratorkonto anlegen
+
+Zuletzt musst du [einen Administrator-Benutzer anlegen](../contao-installtool/#ein-administratorkonto-anlegen), mit dem du dich später im Contao-Backend anmelden kannst.
+
+Nachdem du den Administrator-Benutzer erstellt hast, ist die Installation von Contao abgeschlossen.  
+Der Link am rechten unteren Rand bringt dich zum Backend. Dort legst Du dann
```
Comment (netzarbeiter) [inline docs/manual/installation/quickstart-checklist.de.md]:
```suggestion
Der Link am rechten unteren Rand bringt dich zum Backend. Dort legst du dann
```

File: docs/manual/installation/quickstart.de.md [hunk 7]
```diff
@@ -0,0 +1,110 @@
...
+title: "Contao Installieren – Quickstart"
+description: "Die ersten Schritte zur Contao Installation mit dem Contao Manager."
+url: "installation/quickstart"
+aliases:
+- /de/installation/quickstart/
+weight: 90
+hidden: true
+---
+
+Wir gehen hier davon aus, dass du entweder die aktuellste Version oder die Long Term Support Version mit dem Contao Manager
```
Comment (netzarbeiter) [inline docs/manual/installation/quickstart.de.md]:
```suggestion
Wir gehen hier davon aus, dass du entweder die aktuellste Version oder die [Long Term Support Version](https://docs.contao.org/manual/de/installation/contao-aktualisieren/#long-term-support-versionen) mit dem Contao Manager
```

File: docs/manual/installation/quickstart.de.md [hunk 8]
```diff
@@ -0,0 +1,110 @@
...
+installieren willst. Das ist der einfachste und für Einsteiger empfohlene Weg.
+
+
+## Hosting Konfiguration
+
+Das Hosting konfigurierst Du über das Admin-Panel deines Hosting-Providers.
```
Comment (netzarbeiter) [inline docs/manual/installation/quickstart.de.md]:
```suggestion
Das Hosting konfigurierst du über das Admin-Panel deines Hosting-Providers.
```

File: docs/manual/installation/quickstart.de.md [hunk 9]
```diff
@@ -0,0 +1,110 @@
...
+In Contao befinden sich alle öffentlich erreichbaren Dateien im Unterordner `/public` der Installation. Erstelle
+dazu den Ordner `public` und setze das Wurzelverzeichnis (Document Root) der Installation über das Admin-Panel
+des Hosting-Providers auf diesen Unterordner.
+
+**Beispiel:** `example.com` zeigt auf das Verzeichnis `/www/example/public`
+
+### Datenbank
+
+Contao benötigt für den Betrieb eine Datenbank 
+([MySQL](../../installation/systemvoraussetzungen/#mysql-mindestanforderungen)), die Du idealerweise auch gleich
```
Comment (netzarbeiter) [inline docs/manual/installation/quickstart.de.md]:
```suggestion
([MySQL](../../installation/systemvoraussetzungen/#mysql-mindestanforderungen)), die du idealerweise auch gleich 
```

File: docs/manual/installation/quickstart.de.md [hunk 10]
```diff
@@ -0,0 +1,110 @@
...
+**Benutzername:** Hier legst du den Benutzernamen des Administrators fest.
+
+**Name:** Hier gibst du den Vor- und Nachnamen des Administrators ein.
+
+**E-Mail-Adresse:** Hier gibst du die E-Mail-Adresse des Administrators ein.
+
+**Passwort:** Hier legst du das Passwort des Administrators fest und bestätigst es.
+
+Nachdem du den Administrator-Benutzer erstellt hast, ist die Installation von Contao abgeschlossen.  
+Der Link am rechten unteren Rand bringt dich zum Backend. Dort legst Du dann
```
Comment (netzarbeiter) [inline docs/manual/installation/quickstart.de.md]:
```suggestion
Der Link am rechten unteren Rand bringt dich zum Backend. Dort legst du dann 
```

**Review by netzarbeiter — CHANGES_REQUESTED** [review]
Thank you, Andreas

**Comment by fiedsch** [comment]
Merged changes requested by @netzarbeiter and deleted the `checklist` version (the short version) as more comments favored the long version.

The final step (if agreed upon) would be to redirect to this document when called from the Contao Manager (see [@fritzmg's comment](https://github.com/contao/docs/pull/1199#issuecomment-1555922420))

**Comment by fiedsch** [comment]
@netzarbeiter I don't know, why Github shows "Changes requested" and I don't find any unresolved change requests. Do you see what I am missing? Thank you.

**Review by netzarbeiter — APPROVED** [review]

**Comment by netzarbeiter** [comment]
@fritzmg can you please aprove quickstart.en.md

File: docs/manual/installation/quickstart.en.md [hunk 11]
```diff
@@ -0,0 +1,105 @@
...
+aliases:
+- /en/installation/quickstart/
+weight: 90
+hidden: true
+---
+
+We assume here that you want to install either the latest version or the [Long Term Support Version](https://docs.contao.org/manual/en/installation/update-contao/#long-term-support-versions) with the Contao Manager.
+This is the easiest and recommended way for beginners.
+
+## osting configuration
```
Comment (fritzmg) [inline docs/manual/installation/quickstart.en.md]:
```suggestion
## Hosting configuration
```

File: docs/manual/installation/quickstart.en.md [hunk 12]
```diff
@@ -0,0 +1,105 @@
...
+
+You configure the hosting via the admin panel of your hosting provider.
+
+### Root directory (document root)
+
+In Contao, all publicly accessible files are located in the `/public` subfolder of the installation. Create
+folder `public` and set the root directory (document root) of the installation to this subfolder via the admin panel
```
Comment (fritzmg) [inline docs/manual/installation/quickstart.en.md]:
```suggestion
In Contao, all publicly accessible files are located in the `/public` subfolder of the installation. Create a
folder called `public` and set the root directory (document root) of the installation to this subfolder via the admin panel
```

File: docs/manual/installation/quickstart.en.md [hunk 13]
```diff
@@ -0,0 +1,105 @@
...
+of the hosting provider to this subfolder.
+
+**Example:** `example.com` points to the directory `/www/example/public`.
+
+### Database
+
+Contao requires a [MySQL](../../installation/system-requirements/#mysql-minimum-requirements) database for operation, which you 
+should ideally create right away. The access data will be needed later.
```
Comment (fritzmg) [inline docs/manual/installation/quickstart.en.md]:
```suggestion
should ideally create right away. Its login credentials will be needed later.
```

File: docs/manual/installation/quickstart.en.md [hunk 14]
```diff
@@ -0,0 +1,105 @@
...
+
+
+## Installing the Contao Manager
+
+The [Contao Manager](../../installation/contao-manager/) consists of a single file which can be downloaded from 
+[contao.org](https://contao.org/en/download). After a successful download you will receive a file `contao-manager.phar`.
```
Comment (fritzmg) [inline docs/manual/installation/quickstart.en.md]:
```suggestion
[contao.org](https://contao.org/en/download). After a successful download you will receive a file called `contao-manager.phar`. 
```

File: docs/manual/installation/quickstart.en.md [hunk 15]
```diff
@@ -0,0 +1,105 @@
...
+
+## Installing the Contao Manager
+
+The [Contao Manager](../../installation/contao-manager/) consists of a single file which can be downloaded from 
+[contao.org](https://contao.org/en/download). After a successful download you will receive a file `contao-manager.phar`. 
+Transfer this file to the `public` directory on your web server.
+
+{{% notice info %}}
+`.phar` files are not executed by all hosting providers. For best compatibility, add the `.php` file extension 
+<b>after the upload</b> (rename the file <b>on the server</b> to `contao-manager.phar.php`).
```
Comment (fritzmg) [inline docs/manual/installation/quickstart.en.md]:
```suggestion
<b>after the upload</b> (i.e. rename the file <b>on the server</b> to `contao-manager.phar.php`).
```

File: docs/manual/installation/quickstart.en.md [hunk 16]
```diff
@@ -0,0 +1,105 @@
...
+{{% /notice %}}
+
+## Call Contao Manager
+
+Then call up the URL `www.example.com/contao-manager.phar.php` with your browser, replacing `www.example.com` with your domain.
```
Comment (fritzmg) [inline docs/manual/installation/quickstart.en.md]:
```suggestion
Now call up the URL `www.example.com/contao-manager.phar.php` with your browser, replacing `www.example.com` with your domain.
```

File: docs/manual/installation/quickstart.en.md [hunk 17]
```diff
@@ -0,0 +1,105 @@
...
+You should now see the welcome page of the Contao Manager.
```
Comment (fritzmg) [inline docs/manual/installation/quickstart.en.md]:
```suggestion
You should see the welcome page of the Contao Manager.
```

File: docs/manual/installation/quickstart.en.md [hunk 18]
```diff
@@ -0,0 +1,105 @@
...
+
+![Welcome page of the Contao Manager]({{% asset "images/manual/installation/de/willkommensseite-des-contao-managers.png" %}}?classes=shadow)
```
Comment (fritzmg) [inline docs/manual/installation/quickstart.en.md]: This still needs an English version I think?

File: docs/manual/installation/quickstart.en.md [hunk 19]
```diff
@@ -0,0 +1,105 @@
...
+
+### Basic configuration
+
+Before you install Contao, you need to configure the manager itself. To do this, create a new user by
+assigning a username and password. Username and password are independent of the later Contao installation!
```
Comment (fritzmg) [inline docs/manual/installation/quickstart.en.md]:
```suggestion
Before you install Contao you need to configure the manager itself. To do this create a new user by
assigning a username and password. Username and password are independent of the later Contao installation!
```

I am not an expert on English punctuation - there is likely way more that needs to be changed ;)

**Review by fritzmg — CHANGES_REQUESTED** [review]

File: docs/manual/installation/quickstart.en.md (hunk 18 above)
Comment (netzarbeiter) [reply docs/manual/installation/quickstart.en.md]: @fiedsch can you please provide all English images?

File: docs/manual/installation/quickstart.en.md (hunk 19 above)
Comment (netzarbeiter) [reply docs/manual/installation/quickstart.en.md]: Me neither

**Review by fritzmg — APPROVED** [review]

File: docs/manual/installation/quickstart.en.md (hunk 18 above)
Comment (fiedsch) [reply docs/manual/installation/quickstart.en.md]: Sure. I will try to do that next weekend. Hopefully the weather will not be too good ;-)

**Comment by fiedsch** [comment]
> > @fiedsch can you please provide all English images?
> I will try to do that next weekend.

Just realized I did not have to create any images. I just had to link the respective ones from the existing documentation. So here you go!

**Comment by netzarbeiter** [comment]
@fiedsch thank you, is there a specific reason why this page should not be displayed in the navigation?

**Comment by fiedsch** [comment]
> thank you, is there a specific reason why this page should not be displayed in the navigation?

Yes. The idea was to have a document, that is linked from the Contao download page, where the link currently points to 
`https://docs.contao.org/manual/de/installation/contao-manager/`:

> In der Contao-Dokumentation findest du eine ausführliche [Installationsanleitung](https://docs.contao.org/manual/de/installation/contao-manager/) und kannst nachlesen, welche [Systemvoraussetzungen](https://docs.contao.org/manual/de/installation/systemvoraussetzungen/) dein Server für den Betrieb von Contao erfüllen muss.

The idea discussed on the 2023 Camp that a new User might expect to find a page that (a) ist titled something like "Install Contao" and (b) contains all the information required in one place.

See also the (initial) comment in my first PR https://github.com/contao/docs/pull/1199

In order to make new users see this page,  the link on Contao's download page would have to be changed, or as [@fritzmg's comment](https://github.com/contao/docs/pull/1199#issuecomment-1555922420) suggests a redirect should be established.

Edit: fix typos

**Comment by netzarbeiter** [comment]
I have just created the link [to.contao.org/quickstart](https://to.contao.org/quickstart)

**Pull request merged**